soc/wch/ch32v00x/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2024 Michael Hope
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_CH32V003
+	select RISCV
+	select BUILD_OUTPUT_HEX
+	select RISCV_ISA_RV32E
+	select RISCV_ISA_EXT_ZICSR
+	select RISCV_ISA_EXT_ZIFENCEI
+	select RISCV_ISA_EXT_C
+	select CH32V00X_SYSTICK
```

Comment:
Add `    select ATOMIC_OPERATIONS_C` as the core doesn't have the `A` extension.

This fixes a link time error when using the logging libraries.

Reply:
The `select ATOMIC_OPERATIONS_C` change is pushed now - thanks!

Reply:
@cnlohr Here is the one-liner change I pushed recently.